api/src/test/java/org/pmiops/workbench/api/OfflineUserControllerTest.java

```diff
@@ -44,21 +40,21 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 public class OfflineUserControllerTest extends SpringTest {
   @Autowired private UserService mockUserService;
-  @Autowired private DirectoryService mockDirectoryService;
   @Autowired private OfflineUserController offlineUserController;
+  @Autowired private CloudTasksClient mockCloudTasksClient;
 
+  private List<DbUser> users;
```

Comment:
Codacy says: convert to local.

I'm happy either way.

Reply:
Nice.. didn't know Codacy was capable of this kind of analysis. Done.